styles/buttons.css

```diff
@@ -35,6 +37,140 @@
   color: var(--primary-color);
 }
 
+.link-text-primary {
+  text-decoration: underline var(--primary-color);
+  text-underline-position: under;
+  color: var(--primary-color);
+}
+
+.link-text-primary:hover {
+  color: var(--primary-color-hover);
+  text-decoration-thickness: 3px;
+  font-weight: bold;
+}
+
+/*************** Secondary Button Styles ***************/
+
+.btn-solid-secondary {
+  background-color: var(--secondary-color);
+  color: var(--text-color);
+}
+
+.btn-solid-secondary:hover {
+  background-color: var(--secondary-color-hover);
+}
+
+.btn-outline-secondary {
+  border: 1px solid var(--secondary-color);
+  color: var(--secondary-color);
+}
+
+.btn-outline-secondary:hover {
+  border: 1px solid var(--secondary-color-hover);
+  background-color: var(--secondary-color-hover);
+  color: var(--text-color);
+}
+
 .btn-icon-secondary {
   color: var(--secondary-color);
 }
+
+.link-text-secondary {
+  text-decoration: underline var(--secondary-color);
+  text-underline-position: under;
+  color: var(--secondary-color);
+}
+
+.link-text-secondary:hover {
+  color: var(--secondary-color-hover);
+  text-decoration-thickness: 3px;
+  font-weight: bold;
+}
+
+/*************** Disabled Button Styles ***************/
+
+.btn-disabled {
+  pointer-events: none;
+  opacity: 0.5;
+}
+
+/*************** Buttons with Different Sizes ***************/
+
+.btn-lg-size {
+  font-size: 1.5rem;
+  padding: 1rem 1.5rem;
+}
+.btn-sm-size {
+  font-size: 0.8rem;
+  padding: 0.25rem 0.5rem;
+}
+
+/*************** Floating and Toggle Buttons ***************/
+
+.btn-float {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  position: fixed;
+  bottom: 1rem;
+  right: 1rem;
+  width: 50px;
+  height: 50px;
```

Comment:
Any specific reason to have px?

Reply:
nothing specific, just wanted to have a specific value for the floating button.. Any suggestions in mind? @rohanmathur91

Reply:
rems can be used.